auth-service/src/test/kotlin/rolebased/RoleBased.kt

```diff
@@ -49,7 +49,7 @@ fun Application.roleBasedModule() {
 
     routing {
         authenticate(ACCESS_ADMIN_ONLY) {
-            get("/") {
+            get("") {
```

Comment:
it's required for ktor 1.5.2?

Reply:
https://youtrack.jetbrains.com/issue/KTOR-1615